xml/docu_styleguide.structure.xml

```diff
@@ -1412,12 +1412,17 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
    unfamiliar terms and special jargon.
   </para>
   <para>
-   Define infinitive forms of verbs and singular nouns. Use lowercase for
-   the term unless it is a proper noun.
+   Define infinitive forms of verbs and singular nouns. Do not start the definition with the term itself.
+   Use lowercase for the term unless it is a proper noun.
   </para>
+  <para>Where applicable, group your terms with the <tag class="emptytag">glossdiv</tag>
+   tag for higher ordering.</para>
+  <para>To support automatic alphabetical ordering in localized versions,
+   use <tag class="emptytag">glosslist</tag> and define the style sheet parameter
+   <literal>glossary.sort</literal> as 1. This sorts glossary lists in alphabetical order.</para>
   <para>
    Use cross-references to link acronyms with their written out forms.
-   Define the written out form. Use a <emphasis>See</emphasis> reference for
+   Define the written out form. Use a <emphasis>See also</emphasis> reference for
```

Comment:
One question: DocBook/GeekoDoc provide a `<glosssee/>` tag to do these references consistently. I'd recommend to use them. However, I haven't found it in http://docs.suse.com/style/current/single-html/docu_styleguide/#sec-glossary.

Did we discuss this? Or is it defined somewhere else?

Reply:
We can close this part as this is covered in #270